source/time.tex

```diff
@@ -10417,6 +10417,8 @@
 \end{itemdescr}
 
 \rSec1[time.format]{Formatting}
+\indexlibrary{\idxcode{format}|)}%
+%% See also FIXME below
```

Comment:
Please drop the FIXME; if there is an issue, a github issue is the right procedure to track it.

Reply:
Done. It was partially to serve as a review hint to solicit suggestions for a better indexing range.